app/controllers/director/counselors_controller.rb

```diff
@@ -29,6 +31,9 @@ def create
       )
       if @counselor.save
         puts "COUNSELOR SAVED"
+        @counselor.groups << @group
```

Comment:
No problem! And yeah, that's something I was thinking about as well; I currently did it with the Happy Path to make sure everything was running smooth!